packages/api-admin/src/lib/components/Table/index.tsx

```diff
@@ -0,0 +1,195 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable react-hooks/rules-of-hooks */
+import type {
+    AdminComponentsConfig,
+    AdminTableColumn,
+    AdminTableConfig,
+    EnumsMap,
+} from "@leancodepl/contractsgenerator-typescript-plugin-admin";
+import { assertNotEmpty } from "@leancodepl/contractsgenerator-typescript-utils";
+import { Table as AntTable, TableProps as AntTableProps, TableColumnType, TableProps } from "antd";
+import { ReactNode, useCallback, useMemo, useState } from "react";
+import mkCqrsClient, { CqrsClientConfig } from "../../services/mkReactQueryClient";
```

Comment:
IMO this should import from corelibrary version (not yet published) and not from local copy of react query client

Reply:
Good point however, I'd treat admin a little bit differently - it does need to be perfect from the start. We don't have plans to use it as of now anywhere and we still plan to do some more work on that. So everything there I'd consider subject to refactor 😅